perf/micro/immediate-scheduler/observable/from-array.js

```diff
@@ -8,7 +8,7 @@ module.exports = function fromArray(suite) {
     }
 
     var oldFromArrayWithImmediateScheduler = RxOld.Observable.fromArray(args, RxOld.Scheduler.immediate);
-    var newFromArrayWithImmediateScheduler = RxNew.Observable.fromArray(args);
+    var newFromArrayWithImmediateScheduler = RxNew.Observable.from(args);
```

Comment:
Found this specific test case is now slower than old Rx's fromArray(). Same test for current thread scheduler still performs as expected. (New implementation is faster)